# Fix pod issue after upgrading to MacOS Sonoma and Xcode 15
# https://github.com/status-im/status-mobile/issues/17682

--- ./node_modules/react-native/scripts/ios-configure-glog.sh
+++ ./node_modules/react-native/scripts/ios-configure-glog-patched.sh
@@ -42,9 +42,6 @@
     patch -p1 config.sub fix_glog_0.3.5_apple_silicon.patch
 fi

-export CC="$(xcrun -find -sdk $PLATFORM_NAME cc) -arch $CURRENT_ARCH -isysroot $(xcrun -sdk $PLATFORM_NAME --show-sdk-path)"
-export CXX="$CC"
-
 # Remove automake symlink if it exists
 if [ -h "test-driver" ]; then
     rm test-driver
